core/src/test/scala/matryoshka/spec.scala

```diff
@@ -455,22 +467,28 @@ class FixplateSpecs extends Specification with ScalaCheck with ScalazMatchers {
 
     "cata" should {
       "evaluate simple expr" in {
-        val v = mul(num(1), mul(num(2), num(3)))
-        v.cata(eval) must equal(6)
-        v.convertTo[Mu].cata(eval) must equal(6)
-        v.convertTo[Nu].cata(eval) must equal(6)
+        testRec(
+          mul(num(1), mul(num(2), num(3))),
+          new RecRunner[Exp, Int] {
+            def run[T[_[_]]: Recursive] = _.cata(eval) must equal(6)
+          })
```

Comment:
I think this is a win. You don't have to look at `convertTo`, it's more clear that it's intended to work on _all_ Recursive types, and there's a bit less duplication. If we ever add a fourth recursive type, (I don't know, lazily-evaluated Fix? ;), this will make it so we don't have to touch the tests.

A very minor thing is I would expect the types to appear on `testRec` rather than the runner inside it, since they apply to the first argument, too. That would make the Runner's declaration a little less heavy, which would be help readability, I think. Assuming that works, of course.